docs/guides/scripts.md

```diff
@@ -65,7 +65,7 @@ Now you're ready to run the script:
 
 ```
 $ node scripts/sample-script.js
-Greeter address: 0x5FbDB2315678afecb367f032d93F642f64180aa3
+Greeter deployed to: 0x5FbDB2315678afecb367f032d93F642f64180aa3
```

Comment:
"Greeter address:" is nowhere in `packages/hardhat-core/sample-projects/basic/scripts/sample-script.js`

To be correct, it's actually: "Greeter deployed to:"